### ISSUE TITLE
"metaclass" is not defined

### ISSUE THREAD

**Issue by albireox** [microsoft/pylance-release#328]
## Environment data

-   Language Server version: 2020.9.0
-   OS and version: macOS 11.0
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.5 (pyenv)

## Actual behaviour

Metaclasses raise a warning ``reportUndefinedVariable`` with message ``"metaclass" is not defined``. I think this didn't happen in the past.

## Code Snippet / Additional information

```python
import abc


class BaseClient(metaclass=abc.ABCMeta):
    pass
```

**Comment by erictraut** [comment]
This is the same regression that is reported in #322.

**Comment by albireox** [comment]
This seems solved along with #322. I think it can be closed.
